tools/src/commands/PublishProjectTemplates.ts

```diff
@@ -51,6 +37,45 @@ async function action(options) {
     options.sdkVersion = sdkVersion;
   }
 
+  const sdkTag = `sdk-${semver.major(options.sdkVersion)}`;
+
+  const tagOptions = new Map<string, string[]>();
+  if (semver.prerelease(options.sdkVersion)) {
+    tagOptions.set(`${sdkTag} and next`, [sdkTag, 'next']);
+  } else {
+    tagOptions.set(`${sdkTag} and latest`, [sdkTag, 'latest']);
+  }
+  tagOptions.set(`${sdkTag} and beta`, [sdkTag, 'beta']);
+  tagOptions.set(sdkTag, [sdkTag]);
+
+  const { tagChoice } = await inquirer.prompt<{ tagChoice: string }>([
+    {
+      type: 'list',
+      name: 'tagChoice',
+      prefix: '❔',
+      message: 'Which tags would you like to use?',
+      choices: [...tagOptions.keys(), 'custom'],
+    },
+  ]);
+
+  let tags = tagOptions.get(tagChoice) || [sdkTag];
```

Comment:
it seems like `tagOptions.get(tagChoice)` would only ever be falsey when you select custom, so the `|| [sdkTag]` but might be unnecessary (except to please typescript?)